tools/run_tests/performance/bq_upload_result.py

```diff
@@ -61,16 +61,18 @@ def _upload_netperf_latency_csv_to_bigquery(dataset_id, table_id, result_file):
         sys.exit(1)
 
 
-def _upload_scenario_result_to_bigquery(dataset_id, table_id, result_file):
+def _upload_scenario_result_to_bigquery(dataset_id, table_id, result_file,
+                                        metadata_file):
     with open(result_file, 'r') as f:
         scenario_result = json.loads(f.read())
 
     bq = big_query_utils.create_big_query()
     _create_results_table(bq, dataset_id, table_id)
 
-    if not _insert_result(bq, dataset_id, table_id, scenario_result):
+    if not _insert_scenario_result(bq, dataset_id, table_id, scenario_result,
+                                   metadata_file):
         print('Error uploading result to bigquery.')
-        sys.exit(1)
+    sys.exit(1)
```

Comment:
We should keep the `exit(1)` in if branch, no?

Reply:
You are right the formatter is really not helping here. Thank you.